WcaOnRails/app/models/user.rb

```diff
@@ -462,7 +488,13 @@ def can_manage_teams?
 
   # Returns true if the user can edit the given team.
   def can_edit_team?(team)
-    can_manage_teams? || team_leader?(team)
+    (
+      can_manage_teams? ||
+      team_leader?(team) ||
+
+      # The leader of the WDC can edit the banned competitors list
+      (team == Team.banned && team_leader?(Team.wdc))
+    )
```

Comment:
I prefer using regular boolean operations here.

I had no idea that this would work without brackets! I guess the two options are:

```ruby
    (
      can_manage_teams? ||
      team_leader?(team) ||

      # The leader of the WDC can edit the banned competitors list
      (team == Team.banned && team_leader?(Team.wdc))
    )
```

or:

```ruby
    can_manage_teams? ||
      team_leader?(team) ||

      # The leader of the WDC can edit the banned competitors list
      (team == Team.banned && team_leader?(Team.wdc))
```

I will go with the version without brackets.